libs/classifiers/jetson/face_mask.py

```diff
@@ -1,20 +1,23 @@
 import numpy as np
 import tensorrt as trt
 import pycuda.driver as cuda
-import sys 
+import sys
 import time
 import logging
 import os
 from libs.detectors.utils.fps_calculator import convert_infr_time_to_fps
+from libs.detectors.utils.ml_model_functions import get_model_json_file_or_return_default_values
+
+# logging.getLogger().setLevel(logging.INFO)
+logger = logging.getLogger(__name__)
```

Comment:
I think the logger isn't being used in this file so these lines could be deleted